inductiva/_cli/cmd_resources/_start.py

```diff
@@ -22,7 +22,9 @@ def start_machine_group(args):
 def register(parser):
     """Register the launch of resources commands."""
 
-    subparser = parser.add_parser("start", help="Start computational resources")
+    subparser = parser.add_parser("start",
```

Comment:
I though we had removed this: https://github.com/inductiva/inductiva/pull/1031

Reply:
We did. But not like this. On the loader we ignore files or folders started with "_"